guide/qip/qip-simulator.rst

```diff
@@ -141,17 +161,17 @@ and computation proceeds. To demonstrate, we continue with our previous circuit:
 
 .. testcode::
 
-  from qutip.qip.circuit import ExactSimulator
+  from qutip.qip.circuit import CircuitSimulator
 
-  sim = ExactSimulator(qc, state=zero_state)
+  sim = CircuitSimulator(qc, state=zero_state)
 
 This initializes the simulator object and carries out any pre-computation
 required. There are two ways to carry out state evolution with the simulator.
-The primary way is to use the :func:`qutip.qip.ExactSimulator.run()` and
-:func:`qutip.qip.ExactSimulator.run_statistics()` functions just like before (only
-now with the :class:`qutip.qip.ExactSimulator` class).
+The primary way is to use the :func:`~qutip.qip.circuit.CircuitSimulator.run` and
+:func:`~qutip.qip.circuit.CircuitSimulator.run_statistics` functions just like before (only
```

Comment:
meth meth

Reply:
still not sure if it makes a difference, though - I think I just uniformly used `:obj:` for everything when I was writing Sphinx stuff!

Reply:
By reading to the compiled doc and links I don't see any difference. I don't know if there is anything happening behind it.

Does `:obj:` works for any func, class, meth?

Reply:
I _believe_ Sphinx outputs different CSS classes between `func` and `meth`, but `obj` causes it to look up the type in its database.  Very not certain about that, though.